src/test/java/com/google/cloud/tools/managedcloudsdk/install/InstallerFactoryTest.java

```diff
@@ -45,24 +43,17 @@
   }
 
   @Parameterized.Parameter(0)
-  public OsType OS;
+  public OsType os;
 
   @Parameterized.Parameter(1)
   public Class<? extends InstallScriptProvider> expectedInstallScriptProviderClass;
 
   @Test
   public void testNewInstaller_latestVersion() {
-    Installer installer = new InstallerFactory(Version.LATEST, OS, false).newInstaller(null, null);
-    Assert.assertEquals(LatestInstaller.class, installer.getClass());
+    Installer installer = new InstallerFactory(os, false).newInstaller(null, null);
+    Assert.assertEquals(Installer.class, installer.getClass());
```

Comment:
Now that there is only one Installer type, the class check is unnecessary.

Reply:
hah, I'm dumb.